lib/venice/receipt.rb

```diff
@@ -64,7 +66,7 @@ def to_json
 
     class << self
       def verify(data, options = {})
-        verify!(data, options) rescue false
```

Comment:
This shouldn't be removed. If you look at the `verify!` method, you'll see can still `raise` and error (https://github.com/patrykwojtynski/venice/blob/af437a866b2bce38e8e6ed3f74d454ba229c4845/lib/venice/receipt.rb#L86). This is a fallback to ensure that if `verify!` does `raise`, we don't blow up and instead return `false`. Without it, if `verify!` does `raise` an error, execution would stop and the calling application wouldn't be notified unless they wrapped the call in `begin/rescue`. However, given the name of the method, `verify` without the exclamation point (meaning this method won't `raise` on error), it should fail gracefully and return `false`.